app/src/main/kotlin/io/eugenethedev/taigamobile/ui/components/lists/UserItem.kt

```diff
@@ -33,8 +34,12 @@ import java.time.format.FormatStyle
 @Composable
 fun UserItem(
     user: User,
-    dateTime: LocalDateTime? = null
-) = Row(verticalAlignment = Alignment.CenterVertically) {
+    dateTime: LocalDateTime? = null,
+    navigateToProfile: (userId: Long) -> Unit  = {_ ->}
```

Comment:
I don't understand why I don't need 'userId:Long' because I need to pass it to go to the profile screen:
```
fun NavController.navigateToProfileScreen(id: Long)
    = navigate("${Routes.profile}/$id")
```